coderd/notifications/notifications_test.go

```diff
@@ -869,14 +926,32 @@ func TestNotificationTemplatesCanRender(t *testing.T) {
 				Scan(&titleTmpl, &bodyTmpl)
 			require.NoError(t, err, "failed to query body template for template:", tc.id)
 
-			title, err := render.GoTemplate(titleTmpl, tc.payload, nil)
+			title, err := render.GoTemplate(titleTmpl, tc.payload, defaultHelpers())
 			require.NotContainsf(t, title, render.NoValue, "template %q is missing a label value", tc.name)
 			require.NoError(t, err, "failed to render notification title template")
 			require.NotEmpty(t, title, "title should not be empty")
 
-			body, err := render.GoTemplate(bodyTmpl, tc.payload, nil)
+			body, err := render.GoTemplate(bodyTmpl, tc.payload, defaultHelpers())
 			require.NoError(t, err, "failed to render notification body template")
 			require.NotEmpty(t, body, "body should not be empty")
+
+			partialName := strings.Join(strings.Split(t.Name(), "/")[1:], "_")
```

Comment:
Feels odd to have this in a test which is not explicitly for this purpose.
Can we split this out, rather?

Reply:
ok, let's rely depend on `strings.Split`

Reply:
Oh, sorry you interpreted what I said literally 😆 
I meant splitting out the _functionality_ into another test.